ska_builder.py

```diff
@@ -151,10 +150,13 @@ def build_list_packages():
 def build_all_packages():
     build_list_packages()
 
+with TemporaryDirectory() as tempska:
```

Comment:
Assuming import-less building with setuptools_scm, is this still required?

Reply:
Good point, but I'm not sure how to work with that in general.  This code is forward compatible so it makes sense to me to just merge these updates and cut the SKA handling when we've made the setuptools_scm package updates.

Reply:
Yes, but the code burns a hole in my soul.  We can leave it for now so things work, but it has to come out eventually.  That's why I want to get SCM versioning deployed to flight.

Reply:
I don't get what you guys are discussing here.